plugins/module_utils/_authenticator.py

```diff
@@ -36,6 +37,7 @@ class HashiVaultAuthenticator():
             'jwt',
             'cert',
             'none',
+            'kubernetes',
```

Comment:
When changing the list in `doc_fragments/auth.py`, please change this list to match.